multiuser/permission/che-multiuser-permission-userdevfile/src/main/java/org/eclipse/che/multiuser/permission/devfile/server/UserDevfileDomain.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2012-2018 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.che.multiuser.permission.devfile.server;
+
+import com.google.common.collect.ImmutableList;
+import java.util.List;
+import org.eclipse.che.multiuser.api.permission.server.AbstractPermissionsDomain;
+import org.eclipse.che.multiuser.permission.devfile.server.model.impl.UserDevfilePermissionImpl;
+
+/** Domain for storing devfile's permissions */
+public class UserDevfileDomain extends AbstractPermissionsDomain<UserDevfilePermissionImpl> {
+  public static final String READ = "read";
+  public static final String DELETE = "delete";
+  public static final String UPDATE = "update";
+  public static final String DOMAIN_ID = "userDevfile";
```

Comment:
that's visible thing thought API and as we agreed - use just `devfile` for visible things
```suggestion
  public static final String DOMAIN_ID = "devfile";
```